### ISSUE TITLE
 best model at epoch None step None

### ISSUE THREAD

**Issue by cguetot** [volpato30/DeepNovoV2#3]
Hi there again,

I am trying to create an small model. However I got this message
```

2021-02-11 09:06:40,625 [INFO] __main__: training mode
2021-02-11 09:06:40,626 [INFO] data_reader: input spectrum file: /home/cguetot/projects/epigg/deepnovo/species/models/A3/input_files/spectrum.mgf
2021-02-11 09:06:40,626 [INFO] data_reader: input feature file: /home/cguetot/projects/epigg/deepnovo/species/models/A3/input_files/feature.train.csv
2021-02-11 09:06:40,626 [INFO] data_reader: build spectrum location from scratch
2021-02-11 09:06:43,778 [INFO] data_reader: read 4737 features, 0 skipped by mass, 0 skipped by unknown modification, 0 skipped by length
2021-02-11 09:06:43,778 [INFO] train_func: 148 steps per epoch
2021-02-11 09:06:43,778 [INFO] data_reader: input spectrum file: /home/cguetot/projects/epigg/deepnovo/species/models/A3/input_files/spectrum.mgf
2021-02-11 09:06:43,778 [INFO] data_reader: input feature file: /home/cguetot/projects/epigg/deepnovo/species/models/A3/input_files/feature.valid.csv
2021-02-11 09:06:43,778 [INFO] data_reader: read cached spectrum locations
2021-02-11 09:06:43,785 [INFO] data_reader: read 876 features, 0 skipped by mass, 0 skipped by unknown modification, 0 skipped by length
2021-02-11 09:06:43,811 [INFO] train_func: initialize a set of new parameters
2021-02-11 09:24:43,300 [INFO] train_func: best model at epoch None step None
```

when I try with more data everything seems to be ok.

- What is the error?
- What can I do to fix it?
best, 

Carlos

**Comment by volpato30** [comment]
Please change steps_per_validation in config file to 50. Your training set is very small and contains only has 148 batchs per epoch. The default value of steps_per_validation is 300 which means the validation step will not be run and validation loss won't be evaluated. That's why it prompted "best model at epoch None step None".

**Comment by cguetot** [comment]
as always, thanks so much.

sorry for keep bugging you


*Carlos*
